lua/lang/dart.lua

```diff
@@ -2,12 +2,32 @@ local M = {}
 
 M.config = function()
   O.lang.dart = {
+    flutter_tools = {
+      active = false,
+    },
     sdk_path = "/usr/lib/dart/bin/snapshots/analysis_server.dart.snapshot",
     formatter = {
       exe = "dart",
       args = { "format" },
     },
   }
+  if O.lang.dart.flutter_tools.active then
```

Comment:
After moving the shortcut for this file, the shortcut F doesn't work anymore.

Reply:
@ChristianChiarulli any idea?

Reply:
if I comment "if O.lang.dart.flutter_tools.active then" it would work again.

Reply:
but we need the check...